{"uid":"e6912e4b587f2e2c","name":"test_more_articles_uploads","fullName":"Tests.Automata teszt.test_conduit.TestConduit#test_more_articles_uploads","historyId":"f5bb8eae81f0bbfba83f65ad86575797","time":{"start":1681476674413,"stop":1681476675239,"duration":826},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'Tests/hirek.csv'","statusTrace":"self = <test_conduit.TestConduit object at 0x7f6d33d3e8f0>\n\n    def test_more_articles_uploads(self):\n        manipulatepages = model.ManipulatePages()\n    \n        getusers = model.GetUsers()\n    \n        print()\n        print('ATC 09 - Új adat bevitele')\n        print()\n    \n        # Bejelentkezem az alkalmazásba.\n        getusers.signin(self.browser, 'user1')\n    \n        # Megvárom, amíg megjelenik a \"Log out\" gomb az oldalon.\n        getusers.logout_btn(self.browser)\n    \n        # Feltöltök egy új cikket.\n>       article = manipulatepages.more_articles_uploads_from_data_source(self.browser)\n\nTests/Automata teszt/test_conduit.py:298: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nTests/Automata teszt/model.py:418: in more_articles_uploads_from_data_source\n    self.article_table_reading()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.ManipulatePages object at 0x7f6d33db1840>\n\n    def article_table_reading(self):\n        article_list = []\n    \n>       with open('Tests/hirek.csv', 'r', encoding='UTF-8') as file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'Tests/hirek.csv'\n\nTests/Automata teszt/model.py:257: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_xunit_setup_method_fixture_TestConduit","time":{"start":1681476671324,"stop":1681476674412,"duration":3088},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'Tests/hirek.csv'","statusTrace":"self = <test_conduit.TestConduit object at 0x7f6d33d3e8f0>\n\n    def test_more_articles_uploads(self):\n        manipulatepages = model.ManipulatePages()\n    \n        getusers = model.GetUsers()\n    \n        print()\n        print('ATC 09 - Új adat bevitele')\n        print()\n    \n        # Bejelentkezem az alkalmazásba.\n        getusers.signin(self.browser, 'user1')\n    \n        # Megvárom, amíg megjelenik a \"Log out\" gomb az oldalon.\n        getusers.logout_btn(self.browser)\n    \n        # Feltöltök egy új cikket.\n>       article = manipulatepages.more_articles_uploads_from_data_source(self.browser)\n\nTests/Automata teszt/test_conduit.py:298: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nTests/Automata teszt/model.py:418: in more_articles_uploads_from_data_source\n    self.article_table_reading()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <model.ManipulatePages object at 0x7f6d33db1840>\n\n    def article_table_reading(self):\n        article_list = []\n    \n>       with open('Tests/hirek.csv', 'r', encoding='UTF-8') as file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'Tests/hirek.csv'\n\nTests/Automata teszt/model.py:257: FileNotFoundError","steps":[],"attachments":[{"uid":"c13fcdeca79efbed","name":"stdout","source":"c13fcdeca79efbed.txt","type":"text/plain","size":151}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"_xunit_setup_method_fixture_TestConduit::0","time":{"start":1681476675262,"stop":1681476675319,"duration":57},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"Tests.Automata teszt"},{"name":"suite","value":"test_conduit"},{"name":"subSuite","value":"TestConduit"},{"name":"host","value":"fv-az456-216"},{"name":"thread","value":"3111-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Tests.Automata teszt.test_conduit"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"e6912e4b587f2e2c.json","parameterValues":[]}